### PR TITLE
Und 483n

### PR DESCRIPTION
**Pull request by Ashish-srij**
und 483 new changes ans nav issue fixed

### CODE REVIEW

File: stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.scss [hunk 1]
```diff
@@ -27,40 +28,45 @@
...
+        }
+      }
+    }
     .menu {
       align-items: center;
       display: flex;
       height: 7.1875rem;
       padding-left: 6.9375rem;
-      width: 50%;
+      //width: 50%;
```
Comment (surbhi90) [inline stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.scss:64]: @Ashish-srij please remove commented code

File: stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.scss [hunk 2]
```diff
@@ -27,40 +28,45 @@
...
+      @extend %header-icon;
     }
 
     .icon-close {
       display: none;
-      margin-left: $spacing-12;
+      @include devicebreak(mediumonlytab) {
+        background:url(#{$img-path-Iconfourback}/Times-blue.svg) no-repeat -0.4375rem -0.3125rem/2.125rem 2.125rem;
+      }
+      @extend %header-icon;
```
Comment (surbhi90) [inline stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.scss:42]: @Ashish-srij CountrySiteHeader.scss and GlobalHeader.scss have same code for 'icon-bar' and 'icon-close' classes so can we create placeholder for these code

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments

File: stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.scss (hunk 2 above)
Comment (Ashish-srij) [reply stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.scss]: @surbhi90 done.

File: stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.scss (hunk 1 above)
Comment (Ashish-srij) [reply stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.scss]: @surbhi90 Done

File: stories/Components/Navigationcomponents/Mainnavigation/CountrySiteHeader/CountrySiteHeader.scss [hunk 3]
```diff
@@ -111,17 +111,7 @@
...
-
-      @include devicebreak(mediumonlytab) {
-        display: block;
-      }
-    }
-
-    .icon-close {
-      display: none;
-    }
+    @extend %header-close;
```
Comment (shishirphadke) [inline stories/Components/Navigationcomponents/Mainnavigation/CountrySiteHeader/CountrySiteHeader.scss:114]: @Ashish-srij Change the placeholder name to just `menu-icon`.

File: stories/assets/scss/_mixins.scss [hunk 4]
```diff
@@ -493,4 +493,39 @@
...
+    &:hover{
+      animation: none;
+    }
+    &.show{
+      display: inline-block;
+    }
+  }
+}
+
+%header-close{
```
Comment (shishirphadke) [inline stories/assets/scss/_mixins.scss:513]: @Ashish-srij We can use just one place holder and based on the state (open) change just the background image.

Comment (Ashish-srij) [reply stories/assets/scss/_mixins.scss]: @shishirphadke Done.

File: stories/Components/Navigationcomponents/Mainnavigation/CountrySiteHeader/CountrySiteHeader.scss (hunk 3 above)
Comment (Ashish-srij) [reply stories/Components/Navigationcomponents/Mainnavigation/CountrySiteHeader/CountrySiteHeader.scss]: @shishirphadke Done.

File: stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.jsx [hunk 5]
```diff
@@ -30,8 +30,8 @@ const GlobalHeader = ({
               data={languageswitcherData.languagedata}
               headerText={languageswitcherData.headerText}
             />
-            <IconsBar />
-            <IconsTimesBlue />
+            <a href="javascript:void(0)" className='icon-bar show'>iconbar</a>
+            <a href="javascript:void(0)" className='icon-close'>iconclose</a>
```
[the remark this answers is not in the document]
Comment (Ashish-srij) [reply stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.jsx]: @shishirphadke Done.

File: stories/Components/Navigationcomponents/Mainnavigation/CountrySiteHeader/CountrySiteHeader.jsx [hunk 6]
```diff
@@ -34,13 +34,13 @@ const CountrySiteHeader = ({
...
-          <div className="cell large-3 small-2 top-right">
+          <div className="cell large-3 small-4 top-right">
             <Languageswitcher
               data={languageswitcherData.languagedata}
               headerText={languageswitcherData.headerText}
             />
-            <IconsBar />
-            <IconsTimesBlue />
+            <a href="javascript:void(0)" className='icon-bar show'>iconbar</a>
+            <a href="javascript:void(0)" className='icon-close'>iconclose</a>
```
[the remark this answers is not in the document]
Comment (Ashish-srij) [reply stories/Components/Navigationcomponents/Mainnavigation/CountrySiteHeader/CountrySiteHeader.jsx]: @shishirphadke Done.

**Pull request merged**